tests/cluecode/test_copyrights_fosso.py

```diff
@@ -33,11 +33,12 @@
 import pytest
 
 import cluecode_test_utils
+from commoncode import compat
+from commoncode.system import py3
 from commoncode.testcase import FileDrivenTesting
 from commoncode.text import python_safe_name
 
-
-pytestmark = pytest.mark.scanslow
```

Comment:
Same as above do not remove this marker please.

Reply:
How do I marked that it has been ported?